vignettes/r-markdown.Rmd

```diff
@@ -38,20 +42,20 @@ the `output.var` chunk option. For example:
 rstan::sampling(model)
 ```
 ````
-
-CmdStanR provides a replacement engine, which can be registered as follows:
+## Option 2: Using  CmdStanR for all chunks
+If CmdStanR is being used a replacement engine needs to be registered along the following lines:
 
 ```{r register-engine, message=FALSE}
 library(cmdstanr)
 check_cmdstan_toolchain(fix = TRUE, quiet = TRUE)
 
-register_knitr_engine()
+register_knitr_engine(override = FALSE)
```

Comment:
This should be `override=TRUE` (i.e., override the default of 'stan' chunks using RStan). I will just go ahead and fix this now.